helpers/cache.js

```diff
@@ -16,9 +16,14 @@ module.exports.connect = function (cacheEnabled, config, logger, cb) {
 		return cb(null, { cacheEnabled: cacheEnabled, client: null });
 	}
 
+	Object.keys(config).forEach(function (key){
```

Comment:
according to documentation you don't need to delete fields with nulls
https://github.com/NodeRedis/node_redis
they are replaced with nulls as defaults anyway